### DISCUSSION TITLE
Markdown Cheat Sheet - Cách tạo một file README trên Github với Markdown | TienNHM

### DISCUSSION THREAD

**Discussion by giscus** [TienNHM/tiennhm.github.io#109] [General]
# Markdown Cheat Sheet - Cách tạo một file README trên Github với Markdown | TienNHM

Bạn có thể tạo một file README trên Github với Markdown. Nhưng markdown là gì và cách tạo một file README trên Github với Markdown như thế nào? Hãy cùng tìm hiểu ở bài viết này nhé!

https://tiennhm.github.io/blog/markdown-cheat-sheet

**Comment by TienNHM** [comment]
🧑‍💻❤️
